interfaces/builtin/unity7.go

```diff
@@ -194,6 +194,11 @@ dbus send
 owner @{HOME}/.local/share/mime/**   r,
 owner @{HOME}/.config/user-dirs.dirs r,
 
+# gtk settings
```

Comment:
Similar to how we do it for gsettings, or local mime share or config dir aboves. This is done via symlinks created on the wrapper (desktop-launch)